### PR TITLE
Dependecy and configuration update

### PR DESCRIPTION
**Pull request by jdaly13**
As mentioned in the jira task [165188](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-165188?focusedCommentId=2281841&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2281841) 
This PR is a bug fix that will include the proper dependencies regardless of which dynamic formatjs plugin is required by withIntl.js file
This PR also fixes the babelHelpers issue mentioned in this github [issue](https://github.com/webex/react-widgets/issues/1091)
So this PR should fix these two github issues
https://github.com/webex/react-widgets/issues/1090
and
https://github.com/webex/react-widgets/issues/1091

### CODE REVIEW

**Comment by jdaly13** [comment]
See these two issues regarding babelHelpers as well for context
https://github.com/babel/babel/issues/9366
https://github.com/rollup/rollup/issues/2665

https://babeljs.io/docs/en/6.26.3/babel-plugin-external-helpers#detail

File: packages/node_modules/@webex/webex-widget-base/package.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
   "keywords": [],
   "author": "devsupport@webex.com",
   "license": "MIT",
+  "dependencies": {
```
Comment (adamweeks) [inline packages/node_modules/@webex/webex-widget-base/package.json:10]: Unfortunately, these get overwritten by the build system. If you run `npm run build package.json` you'll see that it no longer has this dependency.

Whatever solution we come up with will need to work with or along side of the [deps.js](https://github.com/adamweeks/react-widgets/blob/e9f321b96216fee09aa0905ab03d29300a211b31/scripts/utils/deps.js#L19) script.

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,7 @@
 npm-debug*
 packages/**/es/*
 packages/**/cjs/*
+packages/**/node_modules/*
```
Comment (adamweeks) [inline .gitignore:10]: Let's scope this change in its own commit.

File: packages/node_modules/@webex/webex-widget-base/package.json (hunk 1 above)
Comment (jdaly13) [reply packages/node_modules/@webex/webex-widget-base/package.json:10]: I updated the deps.js script to look for existing dependencies first and if not just create a simple object

Comment (jdaly13) [reply packages/node_modules/@webex/webex-widget-base/package.json:10]: that should fix the issue

**Comment by jdaly13** [comment]
Hey @adamweeks let me know what you want to do - I have 3 commits.  Do you want me to revert commit 1?  I can possibly change the first commit message but I don't think I can edit that commit (meaning removing the gitignore changes) unless I revert commit.  I think the easiest thing would be to possibly revert the first commit and then do two more commits - one for the original changes with updated commit message and then another commit with the gitignore changes.  I added those gitignore changes because when you add a dependency and do es build it will add node_modules to the package and you have hundreds of new files that would be committed.

**Comment by adamweeks** [comment]
@jdaly13 you can rebase and split the first commit into two in order to get the `gitcommit` changes into their own commit. See this SO question for steps on how to do it: https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits

Also, the last commit message should be `chore(tooling)` for the build process script change.

**Comment by jdaly13** [comment]
@adamweeks I have split up those commits while amending the commit messages as well, this should satisfy requirements.

**Pull request merged**